src/api/AlertIOS.js

```diff
@@ -85,32 +80,24 @@ class AlertIOS {
    */
   static prompt(title, message, callbackOrButtons, type, defaultValue) {
     if (typeof type === 'function') {
-      console.warn(
-        'You passed a callback function as the "type" argument to AlertIOS.prompt(). React Native is ' +
-        'assuming  you want to use the deprecated AlertIOS.prompt(title, defaultValue, buttons, callback) ' +
-        'signature. The current signature is AlertIOS.prompt(title, message, callbackOrButtons, type, defaultValue) ' +
-        'and the old syntax will be removed in a future version.');
-
-      var callback = type;
-      var defaultValue = message;
+      const callback = type;
       AlertManager.alertWithArgs({
         title: title || undefined,
         type: 'plain-text',
-        defaultValue,
+        message,
```

Comment:
i guess `defaultValue` is just never used then? It seems like it wasn't before, either... but i wonder if that's a bug or what?